app/views/layouts/_meta.html

```diff
@@ -0,0 +1,5 @@
+%meta{name: 'title', content: 'hackEDU'}
+%meta{name: 'description', content: 'Be a part of the global movement of free student-led coding clubs.'}
+%meta{name: 'og:type', content: "website"}
+%meta{name: 'og:url', content: "https://hackedu.us"}
+%meta{name: 'og:image', content: "#{ image_path('group_photo_no_gradient.jpg') }"}
```

Comment:
Perhaps `_social_meta.html.haml` would be a better name for this? We have other meta tags that are in `application.html.haml` that shouldn't be in this file (like the viewport declaration).

Also, should the extension of this file end in `.haml`?